### ISSUE TITLE
Accessibility audit

### ISSUE THREAD

**Issue by martinjobb** [Frojd/Frojd-Bedrock#40]
Axe DevTools flagged that "Document does not have a main landmark" when i was testing a site based on Frojd-Bedrock. 

I think the solution would be to replace `<div role="document">` with `<main>` or `<div role="main">` but maybe this should be done together with a bigger accessibility review on the whole project?
